src/control/settings/Settings.cpp

```diff
@@ -739,7 +739,9 @@ void Settings::save() {
     WRITE_STRING_PROP(presentationHideElements);
     WRITE_COMMENT("Which gui elements are hidden if you are in Presentation mode, separated by a colon (,)");
 
-    WRITE_BOOL_PROP(showBigCursor);
+    saveProperty("stylusCursorType", stylusCursorTypeToString(this->stylusCursorType), root);
```

Comment:
The comment gets written on the wrong position in the settings file. Fix:
```suggestion
    xmlNode = saveProperty("stylusCursorType", stylusCursorTypeToString(this->stylusCursorType), root);
```